docs/solutions/sql_server_always_on_availability_groups/modules/instances/Sql/main.tf

```diff
@@ -0,0 +1,54 @@
+resource "oci_core_instance" "instance" {
+  count               = "${var.ad_count}"
+  availability_domain = "${lookup(data.oci_identity_availability_domains.ADs.availability_domains["${count.index}"],"name")}"
+  compartment_id      = "${var.compartment_id}"
+  display_name        = "${var.dns_label}${"${count.index}" + 1}"
+
+  #hostname_label      = "${var.dns_label}${"${count.index}" + 1}"
```

Comment:
remove

Reply:
Remove the hostname_label?

Reply:
It's commented out... is this here in case users want to add it in? If so, just put a comment above it saying so.